src/core/tests/widgets/test_datepicker.py

```diff
@@ -0,0 +1,19 @@
+import toga
+import toga_dummy
+from toga_dummy.utils import TestCase
+
+
+class DatePickerTests(TestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.date_picker = toga.DatePicker(factory=toga_dummy.factory)
+
+    def test_widget_created(self):
+        self.assertEqual(self.date_picker._impl.interface, self.date_picker)
+        self.assertActionPerformed(self.date_picker, 'create DatePicker')
+
+    def test_getting_value_invokes_impl_method(self):
+        self.date_picker.value
```

Comment:
From a code perspective, this *looks* like a no-op. I know it's required to test the getter behavior, but it looks out of place for someone looking at the code. It would be worth adding a comment - something like: "Exercise the value attribute getter; we don't care about the actual value."

(I know there's a couple of other tests on other widgets that don't follow this convention - that's an oversight that we should probably correct)